src/client/components/PageLayout/Toolbar/Status/useRecipients.ts

```diff
@@ -0,0 +1,31 @@
+import { useEffect } from 'react'
+
+import { ApiEndPoint } from '@meta/api/endpoint'
+import { User } from '@meta/user'
+
+import { useAssessment, useCycle } from '@client/store/assessment'
+import { useCountryIso, useGetRequest } from '@client/hooks'
+
+import { StatusTransition } from './types'
+
+export const useRecipients = (props: { status: StatusTransition }): Array<User> => {
+  const { status } = props
+  const countryIso = useCountryIso()
+  const cycle = useCycle()
+  const assessment = useAssessment()
+  const { data, dispatch: fetchData } = useGetRequest(ApiEndPoint.User.countryStatusChangeRecipients(), {
```

Comment:
Any chance we can use users like in country home without adding this new endpoint ?